### PR TITLE
Request forwarding 

### PR DESCRIPTION
**Pull request by bgeuken**
* Implement request forwarding for webui 2
* Fix size of text area fields of comments and request decision box

screenshot_2019-02-06 open build service

### CODE REVIEW

**Comment by bgeuken** [comment]
[Decision Box](https://trello.com/c/vNsJOLOL/1037-decision-box)

File: src/api/app/views/webui2/webui/request/_decision_tab.html.haml [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-    .custom-control.custom-checkbox
-      = check_box_tag('add_submitter_as_maintainer_0', "#{action[:tprj]}_#_#{action[:tpkg]}", false, class: 'custom-control-input')
-      %label.custom-control-label{ for: 'add_submitter_as_maintainer_0' }
-        Add #{link_to(request_creator, user_show_path(request_creator))} as maintainer for
-        #{project_or_package_link(project: action[:tprj], package: action[:tpkg], short: true)}
+    = text_area_tag(:reason, nil, placeholder: 'Please add a comment', rows: 4, class: 'w-100 form-control')
+  - if single_action_request
+    - if state.in?(['new', 'review']) && !action[:creator_is_target_maintainer]
+      .custom-control.custom-checkbox
+        = check_box_tag('add_submitter_as_maintainer_0', "#{action[:tprj]}_#_#{action[:tpkg]}", false, class: 'custom-control-input')
```
Comment (DavidKang) [inline src/api/app/views/webui2/webui/request/_decision_tab.html.haml]: This should be `add_submitter_as_maintainer_#{index}` :thinking:

Comment (bgeuken) [reply src/api/app/views/webui2/webui/request/_decision_tab.html.haml]: This option only works for requests with single actions. That's also the case in the old UI. I just kept the format of the parameter this way so I don't have to change the old code.

**Comment by bgeuken** [comment]
The checkout and backend tests [passed](https://circleci.com/workflow-run/1ee8fca8-4e0c-4e3a-b1ef-f71892b7b24d). It's just not showing up here.

**Comment by coolo** [comment]
now a good time to reenable skipped tests? :)

**Pull request merged**